src/components/graph/graph.config.js

```diff
@@ -200,6 +200,7 @@ export default {
         opacity: 1,
         semanticStrokeWidth: false,
         strokeWidth: 1.5,
-        type: 'STRAIGHT'
+        type: 'STRAIGHT',
+        mouseCursor: 'pointer'
```

Comment:
Please add documentation about this property on jsdoc. You can have as reference the `node.mouseCursor` property jsdoc section `* @param {string} [node.mouseCursor='pointer']`

Reply:
Done in new commit. Please check!